### DISCUSSION TITLE
Integrating WPForms

### DISCUSSION THREAD

**Discussion by stevenwheatley84** [swup/swup#893] [General]
Hi,
So, first of all, absolutely super plugin! Using it for the first time and it's a huge improvement from previous "smooth loading" libraries. Thank you!!!

Am currently creating a WordPress theme and everything works fine apart from...WPForms.

Previously, I've managed to get ContactForm7 to work with a similar library, but my client specifically wants WPForms and I've come to a dead end in my research.

I know that I should be looking for a WPForms function that I can init, but so far I can't find anything.

If anybody can point me in the right direction, it would be hugely appreciated!

**Comment by hirasso** [comment]
You should post your question on the WPForms forums rather than here. It's not related to swup specifically.

You can find more information about hooking into swup's livecycle here:

https://swup.js.org/hooks/#registering-handlers
